src/main/java/bowling/controller/BowlingController.java

```diff
@@ -0,0 +1,30 @@
+package bowling.controller;
+
+import bowling.domain.Frames;
+import bowling.domain.Player;
+import bowling.view.InputView;
+import bowling.view.ResultView;
+
+public class BowlingController {
+
+    private InputView inputView;
+    private ResultView resultView;
+
+    public BowlingController() {
+        this.inputView = new InputView();
+        this.resultView = new ResultView();
+    }
+
+    public void run() {
+        Player player = Player.from(inputView.inputPlayer());
+        Frames frames = Frames.newInstance();
+        frames.init();
```

Comment:
`newInstance()`를 한 뒤 `init()`을 하는 것도 좋지만 
차라리 정적 팩토리 `method `하나로 합치시는건 어떠실까요?? 🤔 
이 부분은 기능상 문제가 있는게 아니라서 검토만 부탁 드리겠습니다 🙇 

```suggestion
        Frames frames = Frames.init();
        Frames frames = Frames.ready(); // 혹은 이와 같이 :D
```